src/components/TodoList.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { TodoElement } from './TodoElement';
+
+type Props = {
+  todos: Todo[] | null,
+  tempTodo?: Todo | null,
+  deleteTodos: (todoId: number) => void,
+  removingId: number | null,
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  tempTodo,
+  deleteTodos,
+  removingId,
+}) => {
+  return (
+    <section className="todoapp__main">
+      {todos?.map(todo => (
+        <TodoElement
+          todo={todo}
+          key={todo.id}
+          deleteTodos={deleteTodos}
+          removingId={removingId}
+        />
+      ))}
+      {tempTodo && (
+        <TodoElement
+          todo={tempTodo}
+        />
```

Comment:
Please add the loader to temp todo
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/54936044/adc7a129-0c86-4790-8ca5-bc4c4963fd53)